lib/main.js

```diff
@@ -102,9 +113,11 @@ async function runElmReview() {
 }
 
 /**
+ * @param {Options} options
+ * @param {ErrorHandler} errorHandler
  * @returns {Promise<void | never>}
  */
-async function runElmReviewInWatchMode() {
+async function runElmReviewInWatchMode(options, errorHandler) {
```

Comment:
I keep meaning to circle back to this. Unrelated to this change, the recursive nature of this function means I think it leaks memory and could potentially stack-overflow. Or just have bad stacktraces.
Related to this diff, because references to `options` & `errorHandler` now need to passed down and stored on the stack of every recursion, I think this change effectively ⅓s the number of saves elm-review can run it watch mode before crashing. It should still be obscenely large, but still a potential issue?

Then again, I could have just misread the code or perhaps v8 does more optimization to recursion than I think it does.